src/main/java/seedu/address/newstorage/JsonRecipeBookStorage.java

```diff
@@ -0,0 +1,79 @@
+package seedu.address.newstorage;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.commons.util.FileUtil;
+import seedu.address.commons.util.JsonUtil;
+import seedu.address.model.ReadOnlyRecipeBook;
+/**
+ * A class to access AddressBook data stored as a json file on the hard disk.
+ */
```

Comment:
minor: update JavaDocs
```suggestion

/**
 * A class to access RecipeBook data stored as a json file on the hard disk.
 */
```